src/domains/analysis/technical/services/price/unified-price.service.ts

```
@@ -49,12 +49,12 @@ export class UnifiedPriceService {
    * @returns Array of price data
    */
   async getHistoricalPrices(
-    platform: Platform,
+    platform: 'paradex' | 'avnu',
     identifier: string,
     timeframe: TimeFrame,
     options: BasePriceOptions = {},
   ): Promise<PriceDTO[]> {
-    const service = this.getPriceService(platform);
+    const service = platform === 'paradex' ? this.paradexService : this.avnuService;
```

If platform is neither paradex or avnu, should throw an error